test_schemas/en/test_list_collector_same_name_items.json

```diff
@@ -315,14 +315,36 @@
                                             "placeholder": "person_name",
                                             "transforms": [
                                                 {
+                                                    "transform": "contains",
                                                     "arguments": {
-                                                        "delimiter": " ",
-                                                        "list_to_concatenate": {
-                                                            "identifier": ["first-name", "last-name"],
-                                                            "source": "answers"
+                                                        "list_to_check": {
+                                                            "source": "list",
+                                                            "id_selector": "same_name_items",
+                                                            "identifier": "people"
+                                                        },
+                                                        "value": {
+                                                            "source": "location",
+                                                            "identifier": "list_item_id"
                                                         }
-                                                    },
-                                                    "transform": "concatenate_list"
+                                                    }
+                                                },
+                                                {
+                                                    "transform": "format_name",
+                                                    "arguments": {
+                                                        "include_middle_name": { "source": "previous_transform" },
```

Comment:
`include_middle_name` should be `include_middle_names` to align with the `middle_names` property.